templates/page.html

```diff
@@ -344,12 +344,24 @@ <h3>Current</h3>
         <div class="card">
           <div class="card-body">
             <h5 class="card-title">
-              <a href="https://github.com/orgs/reanahub/projects/34" class="card-link">
-                User quotas
+              <a href="https://github.com/orgs/reanahub/projects/38" class="card-link">
+                Snakemake integration
               </a>
             </h5>
             <p class="card-text">
-              Collect CPU and HDD consumption information for users running notebooks and workflows.
+              Integrate <a href="https://snakemake.github.io/">Snakemake</a> as a new supported workflow engine language.
```

Comment:
Perhaps remove the trailing word "language" to read "Integrate Snakemake as a new supported workflow engine".

(otherwise we could use "workflow description language" but that's kind of long... plus using just engine plays better with `r-w-e-snakemake)